dobi.yaml

```diff
@@ -287,8 +286,8 @@ image=analyzer-executor-test:
 
 image=analyzer-executor:
   image: grapl/grapl-analyzer-executor
-  context: src/python/analyzer_executor/
-  dockerfile: Dockerfile
+  context: src/python/
```

Comment:
I think ideally the context should be scoped to just the files necessary for the service to run (or build, as the case may be). This context includes the entire python source tree which seems excessive. Might be worth adding a `# hack` comment here too.